lib/rgeo/geographic/spherical_feature_methods.rb

```diff
@@ -128,6 +128,66 @@ def is_simple?
       def length
         arcs.inject(0.0) { |sum, arc| sum + arc.length } * SphericalMath::RADIUS
       end
+
+      def intersects?(rhs)
+        case rhs
+        when Feature::LineString
+          intersects_line_string?(rhs)
+        else
+          super
+        end
+      end
+
+      def crosses?(rhs)
+        case rhs
+        when Feature::LineString
+          crosses_line_string?(rhs)
+        else
+          super
+        end
+      end
+
+      private
+
+      # TODO: replace with better algorithm
```

Comment:
I'm in favor of keeping it as-is, opening an issue if one wants to improve

Reply:
Do you want me to remove the TODOs or just leave them and open an issue?

Reply:
Either remove those or reference the issue in the TODOs, as you wish!

Reply:
Already created #274 which covers this. I'll add that to the TODO